frontend/public/components/utils/storage-class-dropdown.tsx

```diff
@@ -245,4 +251,5 @@ export type StorageClassDropdownInnerProps = {
   filter?: (param) => boolean;
   noSelection?: boolean;
   selectedKey?: string;
+  t: TFunction;
```

Comment:
I don't think this line is needed if you already use `WithTranslation &`?

Reply:
removed